src/services/friends/friends_service.hpp

```diff
@@ -13,6 +13,7 @@ namespace big
 		friends_service& operator=(const friends_service&)     = delete;
 		friends_service& operator=(friends_service&&) noexcept = delete;
 
+		[[nodiscard]] static bool is_friend(int64_t rockstar_id);
```

Comment:
We could probably keep the original and just have an overload for the rid

Reply:
i can declare this one inline